documentation/bolt_types_reference.md

```diff
@@ -93,6 +93,8 @@ The following functions are available to `Target` objects:
 | `protocol` | `String` | The protocol used to connect to the target. | This is equivalent to the target's `transport`, except for targets using the `remote` transport. For example, a target with the URI `http://example.com` using the `remote` transport would return `http` for the `protocol`. |
 | `safe_name` | `String` | The target's safe name. Equivalent to `name` if a name was given, or the target's `uri` with any password omitted. ||
 | `target_alias` | `Variant[String, Array[String]]` | The target's aliases. ||
+| `transport` | `String` | The transport used to connect to the target. ||
+| `transport_config` | `Hash[String, Data]` | The merged configuration for the target's `transport`. | This function returns the merged configuration for a target's `transport`, including defaults, configuration set in a `bolt.yaml`, and configuration set in an `inventory.yaml` file or set in a plan using the `Target.new` or `set_config()` functions. |
```

Comment:
```suggestion
| `transport_config` | `Hash[String, Data]` | The merged configuration for the target's `transport`. | This function returns the merged configuration for a target's transport. This includes defaults, configuration set in a `bolt.yaml`, configuration set `inventory.yaml`, and configuration set in a plan using `set_config()`.|
```

Reply:
I think we can avoid the details about which plan functions override all config here - hopefully anyone using Target.new or set_config will have read the relevant docs.